lib/ui/common/widget/primary_button.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+const _radius = 8.0;
```

Comment:
co Ty na to, żeby przechowywać consty dla danego pliku w abstrakcyjnej klasie _Constants w tym samym pliku jako pola statyczne i odwoływać się do nich wtedy z przedrostkiem `_Constants` ? w ten sposób w kodzie łatwo jest odróżnić consty od reszty pól/metod prywatnych w tym samym pliku